src/commands/basic/prev.js

```diff
@@ -1,13 +1,16 @@
 const { SlashCommandBuilder } = require('@discordjs/builders');
 const { useHistory, useQueue } = require('discord-player');
 const useResume = require('@hooks/useResume');
+const useList = require('@hooks/useList');
+
 module.exports = {
 	data: new SlashCommandBuilder()
 		.setName('prev')
 		.setDescription('Previous track'),
 	async execute({ interaction }) {
 		const history = useHistory(interaction.guild.id);
 		const queue = useQueue(interaction.guild.id);
+		const [, , handleNext] = useList(queue);
```

Comment:
it's better to expose functions like inside an object so u can destruct them conveniently: 
`const { handleNext } = useList(queue);`